authors/apps/articles/tests/test_data.py

```diff
@@ -6,3 +6,23 @@
         "body": "this is the body"
     }
 }
+updated_article = {
```

Comment:
Thanks for the good work @wasibani-roy, however, it looks like an updated article data is similar to valid article data, you could just one them to keep the code DRY.

Reply:
Done